examples/tfengine/generated/full/live/audit/resources/main.tf

```diff
@@ -38,10 +40,11 @@ resource "google_organization_iam_audit_config" "config" {
   }
 }
 
+
 # BigQuery log sink.
 resource "google_logging_organization_sink" "bigquery_audit_logs_sink" {
   name                 = "bigquery-audit-logs-sink"
-  org_id               = var.org_id
+  org_id    = var.org_id
```

Comment:
format

Reply:
Because the template is {{$parent_field}} = {{.parent_id}} this is a bit hard to format correctly. I will let it get handled by the upcoming auto formatting PR.